go.mod

```diff
@@ -72,7 +73,7 @@ require (
 	github.com/signalfx/defaults v1.2.2-0.20180531161417-70562fe60657
 	github.com/signalfx/golib/v3 v3.3.45
 	github.com/signalfx/signalfx-agent v1.0.1-0.20220726135153-ffcbf412e1fa
-	github.com/signalfx/splunk-otel-collector/tests v0.0.0-00010101000000-000000000000
+	github.com/signalfx/splunk-otel-collector/tests v0.0.0-20220803195910-b8b291c66324
```

Comment:
The `replace github.com/signalfx/splunk-otel-collector/tests => ./tests` below makes this unnecessary tmk